module/Company/src/Company/Form/EditCompany.php

```diff
@@ -0,0 +1,325 @@
+            'name' => 'submit',
+            'attributes' => array(
+                'type'  => 'submit',
+                'value' => $translate->translate('Submit changes'),
+                'id' => 'submitbutton',
+            ),
+        ));
+        
+        $this->initFilters();
+
+    }
+    
+    protected function initFilters()
+    {
+        $filter = new InputFilter();
+
+
+        $filter->add(array(
+            'name' => 'name',
+            'required' => true,
+            'validators' => array(
+                array(
+                    'name' => 'string_length',
+                    'options' => array(
+                        'min' => 2,
+                        'max' => 127
+                    )
+                )
+            ),
+            'filters' => array(
+                array('name' => 'StripTags'),
+                array('name' => 'StringTrim')
+            ),
+        ));
+        
+        /*$filter->add(array(
+            'name' => 'en.website',
+            'required' => true,
+            'filters' => array(
+                array('name' => 'StripTags'),
+                array('name' => 'StringTrim')
+            ),
+            'validators' => array(
+            )
+        ));
+        
+        $filter->add(array(
+            'name' => 'nl.website',
+            'required' => true,
+            'filters' => array(
+                array('name' => 'StripTags'),
+                array('name' => 'StringTrim')
+            ),
+            'validators' => array(
+            )
+        ));
+        $filter->add(array(
+            'name' => 'en.description',
+            'required' => true,
+            'validators' => array(
+                array(
+                    'name' => 'string_length',
+                    'options' => array(
+                        'min' => 2,
+                        'max' => 10000
+                    )
+                )
+            )
+        ));
+        $filter->add(array(
+            'name' => 'nl.description',
+            'required' => true,
+            'validators' => array(
+                array(
+                    'name' => 'string_length',
+                    'options' => array(
+                        'min' => 2,
+                        'max' => 10000
+                    )
+                )
+            )
+        ));*/
+        
+        $filter->add(array(
+            'name' => 'email',
+            'required' => true,
+            'validators' => array(
+                array(
+                    'name' => 'EmailAddress',
+                    'options' => array(
+                        'messages' => array(
+                            'emailAddressInvalidFormat' => 'Email address format is not valid'
+                        )
+                    )
+                ),
+            ),
+            'filters' => array(
+                array('name' => 'StripTags'),
+                array('name' => 'StringTrim')
+            ),
```

Comment:
Hier is dit overbodig aangezien er al wordt gevalideerd of iets een email is. En een email adres mag geen html tags bevatten zover ik weet.